### PR TITLE
smimesign 0.0.1 (new formula)

### PR DESCRIPTION
**Pull request by btoews**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/smimesign.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Smimesign < Formula
+  desc "S/MIME signing utility for use with Git"
+  homepage "https://github.com/github/smimesign"
+  url "https://github.com/github/smimesign/archive/0.0.1.tar.gz"
+  sha256 "13789368b1379aa2ad52bce5b855b56ca88f684ed0dff42adf107ee9738187b0"
+  depends_on "go" => :build
```
Comment (mastahyeti) [inline Formula/smimesign.rb]: Can this specify the Go version? This requires Go>=1.10.4.

**Comment by commitay** [comment]
```diff
diff --git a/Formula/smimesign.rb b/Formula/smimesign.rb
index 8798d5d850..b643546acc 100644
--- a/Formula/smimesign.rb
+++ b/Formula/smimesign.rb
@@ -3,14 +3,15 @@ class Smimesign < Formula
   homepage "https://github.com/github/smimesign"
   url "https://github.com/github/smimesign/archive/0.0.1.tar.gz"
   sha256 "13789368b1379aa2ad52bce5b855b56ca88f684ed0dff42adf107ee9738187b0"
+
   depends_on "go" => :build

   def install
     ENV["GOPATH"] = buildpath
     (buildpath/"src/github.com/github/smimesign").install buildpath.children
+
     cd "src/github.com/github/smimesign" do
-      system "go", "build"
-      bin.install "smimesign"
+      system "go", "build", "-o", bin/"smimesign"
       prefix.install_metafiles
     end
   end
```

File: Formula/smimesign.rb (hunk 1 above)
Comment (commitay) [reply Formula/smimesign.rb]: No, it will build with whatever version `go` is, currently `1.11`.

**Comment by fxcoudert** [comment]
Hi @mastahyeti this is a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Comment by mastahyeti** [comment]
@fxcoudert @commitay This tool's repository now meets the notability requirements. Can you reopen this pull request, or would you rather I submit a new one?

**Comment by fxcoudert** [comment]
I can't reopen because your branch has been altered, says github. Please open a new one!

**Pull request closed without merging**